hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -310,15 +310,6 @@
       datanode periodically send container report to SCM. Unit could be
       defined with postfix (ns,ms,s,m,h,d)</description>
   </property>
-  <property>
-    <name>hdds.command.status.report.interval</name>
```

Comment:
Is this a dead key being removed?

Reply:
This property "hdds.command.status.report.interval" appears two times in the ozone-default.xml with different default value. One is 60s, one is 30s. I kept the one with the value (60s) defined in HddsConfigKeys.java, and removed other one.